pkg/model/context.go

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright © 2019 One Concern
+ *
+ */
+
+package model
+
+import (
+	"fmt"
+
+	"gopkg.in/yaml.v2"
+)
+
+const ContextVersion = 1.0
+
+// Context defines the details for a Datamon context.
+type Context struct {
+	Name      string `json:"name" yaml:"name"`           // Name for the context
+	WAL       string `json:"wal" yaml:"wal"`             // WAL is the location for the log
```

Comment:
upon explicit call for further review, i suppose these could be schematized as interfaces